packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -24,6 +24,13 @@ describe('RestServer (integration)', () => {
     await server.stop();
   });
 
+  it('does not throw an error when stopping an app that has not been started', async () => {
+    const app = new Application();
+    app.component(RestComponent);
+    const server = await app.getServer(RestServer);
+    expect(server.stop()).to.not.throw();
```

Comment:
Since `server.stop()` is `async` function, it never throws directly, the function returns a promise that's eventually rejected.

I think you need to rewrite this assertion as follows:

```ts
await expect(server.stop()).to.be.fulfilled();
```

See http://shouldjs.github.io/#assertion-fulfilled